src/System.Net.Sockets/src/System/Net/Sockets/AcceptOverlappedAsyncResult.Windows.cs

```diff
@@ -58,7 +58,7 @@ internal override object PostCompletion(int numBytes)
                         SocketOptionLevel.Socket,
                         SocketOptionName.UpdateAcceptContext,
                         ref handle,
-                        Marshal.SizeOf(handle));
+                        IntPtr.Size);
```

Comment:
why suddenly IntPtr.Size vs. sizeof(IntPtr)? ... both ways seem to be correct, just curious